apis/config/v1beta1/configuration_types.go

```diff
@@ -155,6 +155,21 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// LocalQueueMetrics configures whether metrics should be exposed per LocalQueue
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
```

Comment:
I would like to extend the API by LocalQueueResources to guard the LocalQueue metrics specific to resources, similarly as we guard the metrics for ClusterQueue which use resources: https://github.com/kubernetes-sigs/kueue/blob/cd747db21930fdacbc5f1ecde5512e28030ddef7/pkg/metrics/metrics.go#L309. 

Since we have such a filer for CLusterQueue it makes sense to LocalQueue. The resource-based metrics may have high cardinality. Maybe some users are ok with reporting metrics for all LQs, but not metrics for resources.

Reply:
> I would like to extend the API by LocalQueueResources to guard the LocalQueue metrics specific to resources, similarly as we guard the metrics for ClusterQueue which use resources:

What are the differences between your request and LocalQueueSelector?

Reply:
LocalQueueSelecotr selects LocalQueues but still collects all metrics. My request is to exclude the metrics which have labels for resources, by analogy to what we do for CQs

Reply:
Doesn't it equal to `enable: false`? Current impl does not collect any LQ metrics when the `enable: false`.

Reply:
Yeah, it matches my understanding. SGTM